api/nakadi-event-bus-api.yaml

```diff
@@ -1228,12 +1229,28 @@ definitions:
     properties:
       expected_write_rate:
         type: integer
-      expected_message_size:
+        description: |
+          Write rate for events of this EventType. This rate encompasses all producers of this EventType for
+          a Nakadi cluster.
+
+          Measured in kb/minutes.
+
+      message_size:
         type: integer
-      expected_read_parallelism:
+        description: |
+          Average message size for each Event of this EventType. Includes in the count the whole serialized
```

Comment:
exactly this ambiguity I wanted to avoid. 
to be clear, my intention here is to allow the same value to be produced by internal measurements of nakadi -- blind to the content and interested really on the size of what would be actually written to disk -- and the client. 
this means in practice all of the data *after* enrichment of metadata. 

off course, for setting the default statistics (by the client) can require some extra hops for him to get the value right, but we probably can't measure on our side any differently (nor it would be useful) and we can provide later some tooling for the client to get this numnber right.